bin/test_sphinx.sh

```diff
@@ -1,5 +1,7 @@
 #! /usr/bin/env bash
 
+set -o errexit
```

Comment:
We should make sure all the bash scripts use this. Also `set -x` is useful (it prints what commands are being run).

Reply:
The fact that sane error handling is an opt-in feature in bash never ceases to amaze me.